drivers/hmis/app/graphql/mutations/ce/create_ce_referral.rb

```diff
@@ -42,6 +42,7 @@ def resolve(opportunity_id:, client_id: nil, source_enrollment_id: nil)
           referred_by: current_user,
           client: client,
           source_enrollment: source_enrollment,
+          referral_origin: 'waitlist',
```

Comment:
Suggest to move string to a constant on referral, or (better yet, I think?) making use of a scope to hide the internals of the origin string.
```ruby
scope :originated_from_waitlist, -> { where(referral_origin: 'waitlist') }
...
opportunity.referrals.originated_from_waitlist.create!(...) # no need to specify referral_origin explicitly
```